### ISSUE TITLE
zsh: killed     python export.py

### ISSUE THREAD

**Issue by aPaleBlueDot** [huggingface/swift-transformers#120]
(coreml) hg@007 Mistral7B % python export.py
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:48<00:00, 16.07s/it]
Converting PyTorch Frontend ==> MIL Ops: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████| 5575/5575 [00:02<00:00, 1958.56 ops/s]
Running MIL frontend_pytorch pipeline: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00,  6.52 passes/s]
Running MIL default pipeline: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 79/79 [11:06<00:00,  8.44s/ passes]
Running MIL backend_mlprogram pipeline: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████| 12/12 [00:00<00:00, 17.32 passes/s]
zsh: killed     python export.py

**Comment by aPaleBlueDot** [comment]
Indeed, that's my guess as well, based on two runs ending at different times, where the longer lasting one had more disk space available. Also noticed another GH issue on the same topic where their run lasted much shorter. I'm on an M1 with 16G RAM.

**Comment by aPaleBlueDot** [comment]
Are PyTorch checkpoints required or does safetsensors only HF repos work for the following line?

`torch_model = StatefulMistralForCausalLM(MODEL_ID, max_context_size=max_context_size)`

**Comment by FL33TW00D** [comment]
Responded to this in #123 - closing!
